projects/fusion-e2e/tests/components/alert/alert.pw.ts

```diff
@@ -1,4 +1,4 @@
-import {AlertPage} from './alert-page';
+import {AlertPage} from '../../pages/alert-page';
```

Comment:
Why is this file here? it looks like a test file - if so it should be located under a relevant folder since this is not something you would like to expose in your library

Reply:
You're the one who asked me to move the "pages" out yesterday, aren't you?

Reply:
Separate test to its own folder + change tests file name to a indicative name